jujugui/static/gui/src/app/components/user-profile/test-user-profile.js

```diff
@@ -584,95 +566,20 @@ describe('UserProfile', () => {
         charmstore={charmstore}
         env={env}
         getDiagramURL={sinon.stub()}
-        gisf={false}
         interactiveLogin={true}
         pluralize={sinon.stub()}
         storeUser={sinon.stub()}
         user={users.charmstore} />);
     assert.equal(list.callCount, 2);
   });
 
-  it('requests jem envs if jem is provided and updates state', () => {
```

Comment:
I don't see any tests added here which 'assert' that the `listModels` prop was called with the appropriate values for these three cases.